### ISSUE TITLE
RCS1071 False Positive

### ISSUE THREAD

**Issue by GravlLift** [JosefPihrt/Roslynator#787]
**Product and Version Used**: 
Roslynator 2019 3.1.0 (VS 2019 extension)
Roslynator.Analyzers 3.1.0 (Package)
.NET Core 3.1

**Steps to Reproduce**:

Parameter-less base constructors are recommended for removal as "redundant", even if the base constructor performs logic.

**Actual Behavior**:
```csharp
public class BaseClass
{
    public Guid Id { get; }
    public string Name { get; }

    public BaseClass()
    {
        Id = Guid.NewGuid();
    }

    public BaseClass(string name) : this()
    {
        Name = name;
    }
}

public class ChildClass : BaseClass
{
    public ChildClass() : base() { } // RCS1071: Remove redundant base constructor call

    public ChildClass(string name) : base(name) { }
}
```

**Expected Behavior**:
No RCS1071, removing that base call changes how `ChildClass` is initialized.

**Comment by JosefPihrt** [comment]
Base class constructor will be called even if it's omitted in the derived class constructor declaration.

**Comment by GravlLift** [comment]
@JosefPihrt, it seems you are entirely correct. I appear to be losing my mind this morning, hallucinating behavior that was not actually there.
